lib/utils/utf8.c

```diff
@@ -79,3 +79,46 @@ char *utf8_lcpy(char *dst, const char *src, size_t n)
 
 	return dst;
 }
+
+int utf8_count_chars(const char *s)
+{
+	ssize_t count = 0;
+	const unsigned char *p = (const unsigned char *)s;
+
+	while (*p) {
```

Comment:
No implicit boolean conversions

```suggestion
	while (*p != '\0') {
```